sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/state/metadata/StateMetadataSource.scala

```diff
@@ -17,9 +17,7 @@
 package org.apache.spark.sql.execution.datasources.v2.state.metadata
 
 import java.util
-
```

Comment:
@ericm-db - we need to revert this back to original ?

Reply:
Sorry, inadvertent.